webui/react/src/services/api.ts

```diff
@@ -15,7 +15,28 @@ import {
 import { serverAddress } from 'utils/routes';
 import { isExperimentTask } from 'utils/task';
 
+/* eslint-disable @typescript-eslint/no-var-requires */
+const ndjsonStream = require('can-ndjson-stream');
+
 export const sApi = new DetSwagger.DeterminedApi(undefined, serverAddress());
+export const fetchParamCreator = DetSwagger.DeterminedApiFetchParamCreator();
+
+// FIXME why aren't the types accessible here?
+// DetSwagger.V1TrialLogsResponse;
```

Comment:
@hkang1 any ideas here?  `V1TrialLogsResponse` is explicitly exported in the type definitions in our swagger API package.

Reply:
@hamidzr I noticed that only items with the `declare` prefix in `api.d.ts` are showing up the editor. Can you add `declare` to all the interfaces as well?

Reply:
https://github.com/determined-ai/determined/pull/819  fixes this